### ISSUE TITLE
2FA

### ISSUE THREAD

**Issue by xakod** [CauMoH/WallpaperDownloader#1]
Привет. приложение не дает ввести код для 2FA, сбрасывает обратно окно на главное, не успеваю ввести

**Comment by CauMoH** [comment]
Да, с двухфакторной авторизацией возможны проблемы. Попробую внести фикс.
